server/headless-agent/src/puppeteer/puppetMaster.ts

```diff
@@ -5,6 +5,7 @@
 
 import { EventEmitter } from "events";
 import { IResolvedUrl } from "@fluidframework/driver-definitions";
+import { ITokenClaims } from "@fluidframework/protocol-definitions";
```

Comment:
Dont change this file. You will need to upgrade to latest package and it will probably break other things

Reply:
I reset this file to main. However, did you mean "don't declare `claims` as `ITokenClaims`", rather than don't add additional token claims at all?